from mysql.connector import ProgrammingError
from bd import nova_conexao


